debian/bionic/foreman/control

```diff
@@ -20,7 +20,7 @@ Depends: ruby, ruby2.5, bundler (>= 1.3), zlib1g-dev,
 Recommends: foreman-proxy, foreman-debug (= ${binary:Version}), python-minimal
 Conflicts: ruby-activerecord-deprecated-finders, ruby-foreman-tasks (<< 0.10.1), ruby-sequel-pg
 Replaces: foreman-compute
```

Comment:
Given there's a migration in core that resolves it, you can add it here as well so it's automatically upgraded. We didn't do that for mysql2 because there was no automatic migration.

Reply:
done